www/include/reporting/dashboard/template/viewHostGroupLog.ihtml

```diff
@@ -33,7 +33,7 @@
 	<tr class='list_lvl_3'>
 		<td style='width:300px; padding:5px'>
 			<!-- chart drawing -->
-			{if $id}
+			{if $hostgroup_id}
```

Comment:
```suggestion
			{if isset($hostgroup_id)}
```

Reply:
`{if $hostgroup_id}` could throw a warning in logs if the variable doesn't exist

Reply:
updated